pip/wheel.py

```diff
@@ -780,8 +779,8 @@ def build(self, session, autobuilding=False):
                 if autobuilding:
                     python_tag = pep425tags.implementation_tag
                     # NOTE: Should move out a method on the cache directly.
```

Comment:
This comment can be removed.

What would the right way to implement #4562 with the new API be?